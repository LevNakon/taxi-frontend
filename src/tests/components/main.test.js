import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import Main from '../../components/main';

describe('Main --- test render correctly component', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallow(<Main />)
    });

    it('render correctly component Main', () => {
        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });

});