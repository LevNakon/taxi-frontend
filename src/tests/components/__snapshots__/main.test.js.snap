// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main --- test render correctly component render correctly component Main 1`] = `
<WithStyles(ForwardRef(Grid))
  container={true}
  justify="space-around"
>
  <WithStyles(ForwardRef(Grid))
    item={true}
    md={5}
    xs={10}
  >
    <Connect(SignIn) />
  </WithStyles(ForwardRef(Grid))>
  <WithStyles(ForwardRef(Grid))
    item={true}
    md={5}
    xs={10}
  >
    <WithStyles(ForwardRef(Card))
      className="card_photo"
    >
      <WithStyles(ForwardRef(CardActionArea))>
        <WithStyles(ForwardRef(CardMedia))
          alt="Teleport"
          className="main_photo"
          component="img"
          image="main.png"
          title="Teleport"
        />
        <WithStyles(ForwardRef(CardContent))>
          <WithStyles(ForwardRef(Typography))
            component="h2"
            gutterBottom={true}
            variant="h5"
          >
            Teleport
          </WithStyles(ForwardRef(Typography))>
          <WithStyles(ForwardRef(Typography))
            color="textSecondary"
            component="p"
            variant="body2"
          >
            App which allow you teleport!
          </WithStyles(ForwardRef(Typography))>
        </WithStyles(ForwardRef(CardContent))>
      </WithStyles(ForwardRef(CardActionArea))>
    </WithStyles(ForwardRef(Card))>
  </WithStyles(ForwardRef(Grid))>
</WithStyles(ForwardRef(Grid))>
`;
